app/controllers/stories_controller.rb

```diff
@@ -85,6 +85,13 @@ def create
     end
   end
 
+  def history
+    @story = policy_scope(Story).try(:find, params[:story_id])
```

Comment:
I was not able to guarantee a 400 error on this scenario, so I will keep it as `policy_scope(Story).find_by(id: params[:story_id])`.